### ISSUE TITLE
Feature request: add a copy as cURL command button

### ISSUE THREAD

**Issue by judepereira** [graphql/graphiql#986]
I'm well aware that this is feasible via the dev tools for Chrome/Safari, however, that also copies all the cookies for my domain.

Additionally, Safari, for example, does not allow one to exclude cookies :(

**Comment by benjie** [comment]
I don't think we want this in core as most users won't need it, but I think it would be a good candidate for a plugin in our new plugin system.
